### DISCUSSION TITLE
Change server

### DISCUSSION THREAD

**Discussion by Eirikr70** [immich-app/immich#3956] [Q&A]
Hello you long bearded Immich super experts,
So I had installed Immich on a Raspberry Pi. Since it was quite a bigload for that poor Pi, I decided to migrate to an AMD server. I Rsync'ed the Docker directory and connected the hard drive with the library. I thought I would have found my system just like before. But I saw it asked for a new admin user. So I didn't create it and came to you to ask if I should create that admin with the same id as the former, and then I would find all my system unchanged, or if I should avoid that move and do something else ...
Thank you.

**Comment by ddshd** [comment]
https://documentation.immich.app/docs/administration/backup-and-restore
